src/util/Routes.ts

```diff
@@ -0,0 +1,41 @@
+import IRoute from '../interfaces/Route';
+import Dashboard from "../pages/Dashboard";
+import NewIntegration from "../pages/NewIntegration";
+import Log from "../pages/Log";
+
+const routes: IRoute[] = [
+    {
+        path: '/',
+        name: 'Dashboard',
+        component: Dashboard,
+        exact: true,
+        icon: 'dashboard',
+        navMenu: true
```

Comment:
Maybe is `navMenu` should be `navigationMenu`, or something even more specific? What is it?

Reply:
navMenu is true for the routes to be included in the navBarMenu to the left. Agree, suggesting name change to inNavigationMenu